src/features/duplicates/store.tsx

```diff
@@ -23,6 +23,17 @@ const potentialDuplicatesSlice = createSlice({
   initialState,
   name: 'potentialDuplicates',
   reducers: {
+    duplicateMerged: (state, action: PayloadAction<number>) => {
+      const mergedDuplicateId = action.payload;
+      const potentialDuplicatesFiltered =
+        state.potentialDuplicatesList.items.filter(
+          (item) => item.id !== mergedDuplicateId
+        );
+
+      if (potentialDuplicatesFiltered) {
```

Comment:
This `if` statement should be unnecessary. As far as I can tell, it's always truthy because `Array.prototype.filter()` always returns an array.